index.js

```diff
@@ -63,8 +63,13 @@
 //.
 //. The next step is to define a `def` function for the environment:
 //.
+//# create :: { checkTypes :: Boolean, env :: Array Type } -> Module
+//.
 //. ```javascript
-//. const def = $.create ({checkTypes: true, env});
+//. const def = $.create ({
+//.   checkTypes: true,
+//.   env: env,
+//. });
```

Comment:
`{checkTypes: true, env}` is equivalent to `{checkTypes: true, env: env}` in modern environments, so I don't think this change is necessary.

Reply:
sounds good.